README.md

```diff
@@ -484,7 +491,11 @@ assuming your kernel is configured to work that way.
 u-root can create an initramfs in two different modes, specified by `-build`:
 
 *   `bb`/`gbb` mode: One busybox-like binary comprising all the Go tools you ask to
-    include. See [here for how it works](pkg/bb/README.md).
+    include, which can be [templates](#templates) and/or individual commands.
+    See [the gobusybox README for how it works](https://github.com/u-root/gobusybox).
+    **NOTE**: `gbb` is now the default variant. `bb` is an older implementation
```

Comment:
How about we just stop referring to bb from now on?

Reply:
yea, good point, I'll rewrite it and drop it